contracts/ccip/GovernanceCCIPRelay.sol

```diff
@@ -69,28 +46,18 @@ contract GovernanceCCIPRelay {
     destinationReceiver = _receiver;
   }
 
-  /**
-   * @notice Sets the address of the receiver contract on the destination chain.
-   * @param _receiver The address of the receiver contract on the destination chain.
-   * @dev Only the Timelock contract can call this function.
-   */
+  /// @inheritdoc IGovernanceCCIPRelay
   function setDestinationReceiver(address _receiver) external onlyTimeLock {
```

Comment:
@voith it's not early returning as suggested in the finding https://cantina.xyz/code/8b6d2575-95bc-4380-8f2a-314e44776822/findings/4

Reply:
Lets discuss this in https://github.com/cryptexfinance/contracts/pull/169#discussion_r1967712065